Makefile

```diff
@@ -1,4 +1,4 @@
-install: composer assets
+install: composer assets docker-override
```

Comment:
Why add this command on `make install` ? If I want to install PrestaShop locall by using `make install`, I don't want to use docker.

Reply:
This does not start any docker container.

The docker-compose.override file will be added when installing the development environment.

This is a prerequisite for using docker later. But I can remove it, `Make docker` command is enough

Reply:
even it doesn't start any container, I will have an error if docker is not installed on my computer. I think it should be removed.

Reply:
There would have been no errors, it is a command `cp docker-compose.override.yml.dist docker-compose.override.yml` which would have been launched. But I removed it

Reply:
you're right about the error 😉 but I think it's better to not have `docker` things during `make install`

Reply:
I agree :smiley: